### ISSUE TITLE
Slow backups from production

### ISSUE THREAD

**Issue by croaky** [thoughtbot/parity#94]
Working on a ~500MB database. It is slow to backup through `development restore production`.

This alternative may be faster:

```
curl -O tmp/latest.backup $(production pg:backups public-url -q)
pg_restore tmp/latest.backup
```

Not sure why yet.

**Comment by geoffharcourt** [comment]
@croaky did you continue to see this issue? Working on Parity today and can target it if it's still a concern.

**Comment by croaky** [comment]
I think it's still an issue, yes.

**Comment by geoffharcourt** [comment]
Implemented in 80eafa3587076ec73c6fdc4b8ce02fddbb5855ad!
